src/content/docs/extensions/algos/wcc.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Weakly Connected Components
+---
+
+A weakly connected component (WCC) in a directed graph is a maximal subgraph where any two vertices
```

Comment:
I feel we lack some context for non-advanced users here:
- How is WCC different from SCC?
- What are some practical applications of WCC?

Reply:
I don't know if we should make it that detailed for this first release. As we flesh out the extension more and we add more and more algorithms, we can add more detail to the use cases and provide more recommendations. For this release, I think it's enough to maybe differentiate between WCC and SCC.

Reply:
I think we can expect users to have a general sense of the algorithms if they're looking into a graph algorithsm library.

Instead of repeating information, we can just link out to Wikipedia or other resources.